src/packages/segmented/segmented.tsx

```
@@ -0,0 +1,80 @@
+    defaultValue: mergedProps.defaultValue,
+    finalValue: 0,
+    onChange: mergedProps.onChange,
+  })
+  const renderItems = useCallback(
+    (options: SegmentedProps['options'], value: string | number) => {
+      return options.map((option, index) => {
+        const optionType = typeof option
+        switch (optionType) {
+          case 'object':
+            // eslint-disable-next-line no-case-declarations
+            const opt = option as SegmentedItem
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)